src/cls/journal/cls_journal_types.h

```diff
@@ -12,6 +12,10 @@
 #include <set>
 #include <string>
 
+namespace {
+static const uint64_t JOURNAL_MAX_RETURN = 256;
```

Comment:
Minor: Shouldn't have anonymous namespaces in headers. You can move this to the cls::journal namespace.